aiapy/calibrate/util.py

```diff
@@ -12,40 +11,119 @@
-# This needs to be incremented as the calibration is updated in JSOC.
-CALIBRATION_VERSION = 10
 # Error table filename available from SSW
-AIA_ERROR_FILE = "sdo/aia/response/aia_V{}_error_table.txt"
-# Most recent version number for error tables; increment as new versions become available
-ERROR_VERSION = 3
+_AIA_ERROR_FILE = "sdo/aia/response/aia_V{}_error_table.txt"
 # URLs and SHA-256 hashes for each version of the error tables
-URL_HASH = {
+_URL_HASH_ERROR_TABLE = {
     2: (
-        [urljoin(mirror, AIA_ERROR_FILE.format(2)) for mirror in _SSW_MIRRORS],
+        [urljoin(mirror, _AIA_ERROR_FILE.format(2)) for mirror in _SSW_MIRRORS],
         "ac97ccc48057809723c27e3ef290c7d78ee35791d9054b2188baecfb5c290d0a",
     ),
     3: (
-        [urljoin(mirror, AIA_ERROR_FILE.format(3)) for mirror in _SSW_MIRRORS],
+        [urljoin(mirror, _AIA_ERROR_FILE.format(3)) for mirror in _SSW_MIRRORS],
         "66ff034923bb0fd1ad20e8f30c7d909e1a80745063957dd6010f81331acaf894",
     ),
 }
+_URL_HASH_POINTING_TABLE = (
+    "https://aia.lmsal.com/public/master_aia_pointing3h.csv",
+    "a2c80fa0ea3453c62c91f51df045ae04b771d5cbb51c6495ed56de0da2a5482e",
+)
+_URL_HASH_RESPONSE_TABLE = {
+    10: (
+        [urljoin(mirror, "sdo/aia/response/aia_V10_20201119_190000_response_table.txt") for mirror in _SSW_MIRRORS],
+        "0a3f2db39d05c44185f6fdeec928089fb55d1ce1e0a805145050c6356cbc6e98",
+    ),
+    9: (
+        [urljoin(mirror, "sdo/aia/response/aia_V9_20200706_215452_response_table.txt") for mirror in _SSW_MIRRORS],
+        "f24b384cba9935ae2e8fd3c0644312720cb6add95c49ba46f1961ae4cf0865f9",
+    ),
+    8: (
+        [urljoin(mirror, "sdo/aia/response/aia_V8_20171210_050627_response_table.txt") for mirror in _SSW_MIRRORS],
+        "0e8bc6af5a69f80ca9d4fc2a27854681b76574d59eb81d7201b7f618081f0fdd",
+    ),
+    7: (
+        [urljoin(mirror, "sdo/aia/response/aia_V7_20171129_195626_response_table.txt") for mirror in _SSW_MIRRORS],
+        "ac2171d549bd6cc6c37e13e505eef1bf0c89fc49bffd037e4ac64f0b895063ac",
+    ),
+    6: (
+        [urljoin(mirror, "sdo/aia/response/aia_V6_20141027_230030_response_table.txt") for mirror in _SSW_MIRRORS],
+        "11c148f447d4538db8fd247f74c26b4ae673355e2536f63eb48f9a267e58c7c6",
+    ),
+    4: (
+        [urljoin(mirror, "sdo/aia/response/aia_V4_20130109_204835_response_table.txt") for mirror in _SSW_MIRRORS],
+        "7e73f4effa9a8dc55f7b4993a8d181419ef555bf295c4704703ca84d7a0fc3c1",
+    ),
+    3: (
+        [urljoin(mirror, "sdo/aia/response/aia_V3_20120926_201221_response_table.txt") for mirror in _SSW_MIRRORS],
+        "0a5d2c2ed1cda18bb9fbdbd51fbf3374e042d20145150632ac95350fc99de68b",
+    ),
+    2: (
+        [urljoin(mirror, "sdo/aia/response/aia_V2_20111129_000000_response_table.txt") for mirror in _SSW_MIRRORS],
+        "d55ccd6cb3cb4bd1c688f8663f942f8a872c918a2504e5e474aa97dff45b62c9",
+    ),
+}
+
+
+def _fetch_response_table(version: int):
```

Comment:
It's confusing that some places this is referred to as the "response table" and some places as the "correction table".

Reply:
Renamed to correction tables.